contents/docs/session-replay/privacy.mdx

```diff
@@ -0,0 +1,99 @@
+---
+title: Privacy Controls
+---
+
+PostHog offers a range of controls to limit what data is captured. 
+
+
+## Inputs
+
+As inputs are highly likely to contain sensitive text such as emails or passwords, **we mask these by default**. You can set this explicitly to false to enable the recording of inputs.
+
+### Mask all inputs
+
+```ts
+posthog.init('<ph_project_api_key>', {
+    session_recording: {
+        // This is the default
+        maskAllInputs: true
+    }
+})
+```
+
+### Mask or un-mask specific inputs
+
+You can control the masking more directly by using the `maskInputFn` to customize how the masking behaves. For example, you may want to only redact text that looks like an email address or that comes from inputs that aren't search boxes.
+
+```ts
+posthog.init('<ph_project_api_key>', {
+    session_recording: {
+        maskAllInputs: true, // Important - this needs to be true for the below function to be called
+        maskInputFn: (text, element) => {
+            if (element?.attributes['type']?.value === 'search') {
+                // If this is a search input, don't mask it
+                return text
+            }
+            // Otherwise, mask it with asterisks
+            return '*'.repeat(text.length)
+        },
+    }
+})
+```
+
+## Text
+
+General text on the website is not masked by default but we provide multiple options for masking text.
+
+### Mask all text
+
+```ts
+posthog.init('<ph_project_api_key>', {
+    session_recording: {
+        maskTextSelector: "*" // Masks all text elements
+    }
+})
+```
+
+### Mask or un-mask specific text
+
+You can use a CSS selector to mask specific elements. For example, you may want to mask all elements with the class `password` or the ID `sensitive`.
+
+```ts
+posthog.init('<ph_project_api_key>', {
+    session_recording: {
+        maskTextSelector: ".password #sensitive" // masks all elements with the class "password" or the ID "sensitive"
```

Comment:
```suggestion
        maskTextSelector: ".password, #sensitive" // masks all elements with the class "password" or the ID "sensitive"
```

isn't the comma `OR` otherwise CSS `AND`s the selectors?

Reply:
You are correct!